wip-0019.md

```diff
@@ -0,0 +1,182 @@
+<pre>
+  WIP: WIP-0019
+  Layer: Consensus (hard fork)
+  Title: Random Number Generation (RNG) functionality
+  Authors: Adán SDPC <adan@witnet.foundation>
+  Discussions-To: https://github.com/witnet/witnet-rust/discussions/2058
+  Status: Draft
+  Type: Standards Track
+  Created: 2021-09-06
+  License: BSD-2-Clause
+</pre>
+
+
+## Abstract
+
+A new type of data source is proposed such that leverages the crowd-attestation mechanism of the Witnet protocol
```

Comment:
The **such that leverages** sounds weird to me. Maybe something like this?

```suggestion
A new type of data source is proposed to leverage the crowd-attestation mechanism of the Witnet protocol to enable Random Number Generation (RNG) functionality inside data requests.
```

Reply:
Too British for a WIP, my bad! :rofl: